bot/connection_pool/__init__.py

```diff
@@ -34,14 +34,14 @@ class Unhealthy(Expired):
     """Connection was unhealthy"""
 
 
-T = TypeVar('T')
+_T = TypeVar('_T')
```

Comment:
Why this change?

Reply:
Plyint wanted it to be a private variable.